package.json

```diff
@@ -26,13 +29,32 @@
   "scripts": {},
   "maintainers": [
     {
-      "name": "Eli Perelman",
+      "name": "eliperelman",
       "email": "eli@eliperelman.com"
     },
     {
-      "name": "Rob Wood",
+      "name": "rwood-moz",
       "email": "rwood@mozilla.com"
     }
   ],
-  "directories": {}
+  "directories": {},
+  "gitHead": "6e9ddcd5b7db2ada4bba417fcca0b897849444a7",
+  "bugs": {
+    "url": "https://github.com/eliperelman/node-mozdevice/issues"
+  },
+  "homepage": "https://github.com/eliperelman/node-mozdevice",
+  "_id": "mozdevice@1.2.1",
+  "_shasum": "a3c5cd264efd1b8be8abf3d80e0009346cd14282",
+  "_from": "mozdevice@>=1.2.1 <2.0.0",
+  "_npmVersion": "2.1.9",
+  "_nodeVersion": "0.10.33",
+  "_npmUser": {
+    "name": "eliperelman",
+    "email": "eli@eliperelman.com"
+  },
+  "dist": {
+    "shasum": "a3c5cd264efd1b8be8abf3d80e0009346cd14282",
+    "tarball": "http://registry.npmjs.org/mozdevice/-/mozdevice-1.2.1.tgz"
```

Comment:
Should this be mozdevice-1.3.0.tgz? Although it doesn't exist until this ver is published...